crates/graphql/src/schema/objects/ah_offer.rs

```diff
@@ -36,6 +36,15 @@ impl Offer {
         &self.buyer
     }
 
+    async fn buyer_wallet(&self, ctx: &AppContext) -> Wallet {
+        let twitter_handle = ctx
+            .twitter_handle_loader
+            .load(self.buyer.clone())
+            .await
+            .unwrap_or_default();
```

Comment:
Is the unwrap_or_default needed? Wallet second parameter already accepts Option<String> which is the return of the loader.

Reply:
Yes, otherwise it will send a result type and give the following compile time error:

```rust
mismatched types
expected enum `std::option::Option<_>`
   found enum `std::result::Result<std::option::Option<_>, batcher::Error>`
```